content/200-concepts/100-components/02-prisma-client/056-composite-types.mdx

```diff
@@ -90,6 +86,47 @@ There are currently some limitations when using composite types in the Prisma Cl
 - [`findUnique`](/reference/api-reference/prisma-client-reference#findunique) can't filter on composite types
 - [`aggregrate`](/concepts/components/prisma-client/aggregation-grouping-summarizing#aggregate), [`groupBy`](/concepts/components/prisma-client/aggregation-grouping-summarizing#group-by), [`count`](/concepts/components/prisma-client/aggregation-grouping-summarizing#count) don’t support composite operations
 
+## Default values on required composite types
+
+From v4.0.0, if you carry out a database read on a composite type when all of the following conditions are true, then Prisma Client inserts the default value into the result.
+
+Conditions:
+
+- The composite type is [required](/concepts/components/prisma-schema/data-model#optional-and-mandatory-fields), and
+- the composite type has a [default value](/concepts/components/prisma-schema/data-model#defining-a-default-value), and
+- the database read operation returns the value `undefined`.
```

Comment:
This is technically incorrect, mongo can't return `undefined`, the field is simply not present on the returned document(s) (aside: `undefined` is a state property of the data, not a data value even if JS looks like it treats it as one).